test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -778,9 +778,9 @@ public void RemoveVariablesInPipelineTest()
                     var onnxEstimator = mlContext.Transforms.ApplyOnnxModel(onnxModelPath);
                     var onnxTransformer = onnxEstimator.Fit(data);
                     var onnxResult = onnxTransformer.Transform(data);
-                    CompareSelectedColumns<float>("Score", "Score", transformedData, onnxResult);
-                    CompareSelectedColumns<float>("Probability", "Probability", transformedData, onnxResult);
-                    CompareSelectedColumns<bool>("PredictedLabel", "PredictedLabel", transformedData, onnxResult);
+                    CompareResults("Score", "Score", transformedData, onnxResult, rightColumnFromOnnx: true);
```

Comment:
>rightColumnFromOnnx [](start = 82, length = 19)

Isn't the right column in all these tests in this file always from Onnx? So why do only a few tests have rightColumnFromOnnx set to true?

Reply:
That is a good point. Maybe you can help me think of a better name. Basically, that value is supposed to be set to true if the right column is from ONNX and its a non vector value. Onnx always returns a vector, even when its a scalar, so this flag handles that case. It grabs a vector instead of the scalar and just gets index 0 from it.

Reply:
Would `isOnnxScalar` work?

---
In reply to: [437556818](https://github.com/dotnet/machinelearning/pull/5205#discussion_r437556818) [](ancestors = 437556818)

Reply:
That is more clear. Maybe `IsRightColumnOnnxScalar` better since its specifically the right column that is the onnx comparison. I'll change it to that.

Reply:
Renamed.